cypress/utils/commands.js

```diff
@@ -331,7 +337,7 @@ Cypress.Commands.add('deleteSavedObjectByType', (type, search) => {
 Cypress.Commands.add('createIndexPattern', (id, attributes, header = {}) => {
   const url = `${
     Cypress.config().baseUrl
-  }/api/saved_objects/index-pattern/${id}`;
+  }/api/saved_objects/index-pattern/${id}?overwrite=true`; // When running tests locally if ran multiple times the tests fail. The fix is to set Overwrite to true.
```

Comment:
Is the test not cleaning data after each run? This seems odd to me. A test case should setup and teardown to not affect other test runs.

Reply:
Did it fail locally multiple times for the new tests or for all tests? If it was because a previous test failed to succeed and kept stale data since it didn't properly clean up. Then I think we can remove this and ensure before tests cleaned up prior to test run. Otherwise make the overwrite an optional param and default to false incase we wanted to have a test that verifies conflicts can't be added unless it's intended.

Reply:
I'm working on upgrading this repo to Cypress12 now and have noticed a few test suites that are not properly isolated.

Any new PRs should be properly isolating each test case because it will be enforced in the upgraded version of Cypress.

Reply:
Completed